linter/yamllint_rules.yml

```diff
@@ -1,10 +1,26 @@
 extends: default
 rules:
-  document-start: disable
+  document-start:
+    level: error
+    present: false
+  document-end:
+    level: error
+    present: false
+  empty-values:
+    forbid-in-block-mappings: true
+    forbid-in-flow-mappings: true
   line-length: disable
   indentation:
-    level: warning
+    level: error
   new-line-at-end-of-file:
     level: error
   trailing-spaces:
     level: error
+  comments:
```

Comment:
Yes, it makes sense. Most of comments in conandata.yml are only to skip some build. Doing a quick search, googleapis, libsodium, imgui and few others have some useful comment.

Reply:
I think this good... I see a handful of comments on the patches which would be nice to role into the new metadata 

There are a very few cases where this might be painful... some have notes about the version for instance 🤷 seems odd

Reply:
This rule is just about blanks before comments, this is not about allowing (or not) comments: https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.comments